modules/Bio/EnsEMBL/Compara/PipeConfig/UpdateReferenceDatabase_conf.pm

```diff
@@ -50,10 +50,6 @@ sub default_options {
         'ref_db'       => 'compara_references',
         'taxonomy_db'  => 'ncbi_taxonomy',
 
-        # Back compatibility for production team's use of '--pass' instead of '--password'
-        'pass'     => undef,
-        'password' => $self->o('pass') ? $self->o('pass') : $self->o('password'),
-
```

Comment:
If this is to support Production, we could leave just in case they take it over again in the future?

Reply:
No, it's really annoying for us, and it doesn't sound like Production will ever take it back :innocent: